Java/core/src/main/java/com/amazon/randomcutforest/RandomCutForest.java

```diff
@@ -240,9 +240,9 @@ public RandomCutForest(Builder<?> builder) {
     }
 
     private void initCompactDouble() {
-        int maxupdate = 1;
+        int maxUpdate = 1;
```

Comment:
Right now we are performing one update at a time. Thus we need exactly one additional space in point store. If for some reason we perform multiple ones (out of order shingled data, for example) then this would have to be changed. In general the three parts show how the capacity was arrived at. If we want to perform dynamic resizing, we may want viz into these different pieces.